frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -18,65 +20,53 @@ export const scene: SceneExport = {
 const OnboardingWrapper = ({ children }: { children: React.ReactNode }): JSX.Element => {
     const { onboardingStep } = useValues(onboardingLogic)
     const { setTotalOnboardingSteps } = useActions(onboardingLogic)
+    const { product } = useValues(onboardingLogic)
+    const [allSteps, setAllSteps] = useState<JSX.Element[]>([])
```

Comment:
Could this be moved into the `onboardingLogic` and then `setTotalOnboardingSteps` could be a computed property on the logic rather than needing to be set e.g:

```
totalOnboardingSteps: [
    (s) => [s.allSteps],
    (allSteps) => Array.isArray(allSteps) ? allSteps.length - 1 : 1,
],
```

Reply:
Yep, I'm changing this a bunch in upcoming PR so will do that then 👍